setup.py

```diff
@@ -16,10 +16,13 @@
     packages=['django_opentracing', 'tests'],
     platforms='any',
     install_requires=[
-        'django<2',
+        'django',
```

Comment:
I've found that life is easier if you don't put Django in `install_requires` at all. Similar to npm's peer dependencies concept. It won't work without Django, but no one is going to add `django_opentracing` to their `requirements.txt` and accept whatever version of Django it brings along. If there are version limitations, they can go into the documentation like [django-reversion](https://django-reversion.readthedocs.io/en/stable/django-versions.html)

Reply:
This seems not to be the OT-contrib convention for other instrumentations, but can potentially lead to problems.  @carlosalberto, do you have any preference or thoughts?

Reply:
Well, usually in your application you have a dependency to `django` too (most likely with a specific version), and not a single dependency to `django_opentracing`.

On the other hand, I think we could put `django` in the test-requires section of `setup.py`, so the tests work just fine - but that would need to happen in a different PR/issue, so we can try it out (and later probably use this defensive behavior in other repos).

Makes sense?